src/routes/Send.tsx

```diff
@@ -569,10 +553,15 @@ export function Send() {
                     sentDetails.fee_estimate = payment?.fees_paid || 0;
                 }
             } else if (source() === "lightning" && lnurlp()) {
+                const zapNpub =
+                    visibility() === "anonzap" && contact()?.npub
+                        ? contact()?.npub
+                        : undefined;
+                console.log("zapnpub", zapNpub);
                 const payment = await state.mutiny_wallet?.lnurl_pay(
                     lnurlp()!,
                     amountSats(),
-                    undefined, // zap_npub
+                    zapNpub, // zap_npub
                     tags,
                     undefined // comment
```

Comment:
if it's an anon zap, should we set the comment to `whatForInput()`?

Reply:
I added this but I don't know how to test if the note is getting through